src/components/CaptureFilter.js

```diff
@@ -338,7 +252,7 @@ function Filter(props) {
               <TextField
                 htmlFor="grower-identifier"
                 id="grower-identifier"
-                label="Grower Identifier"
+                label="Wallet"
                 placeholder="e.g. grower@example.com"
                 value={growerIdentifier}
```

Comment:
Does this need to become `wallet`?

Reply:
I figure we're going to get confused if we don't call it the same thing in the UX as we do in the database, but the rest didn't have to change because it is only visible in the code.  Eventually we can change all the form references if needed.